File: SDKs/apicurio.local/registry/2.1.0-SNAPSHOT/typescript/src/sdk/models/operations/exportdata.ts
import { SpeakeasyMetadata, SpeakeasyBase } from "../../../internal/utils";
import * as shared from "../shared";



export class ExportDataResponse extends SpeakeasyBase {
  @SpeakeasyMetadata()
  contentType: string;

  @SpeakeasyMetadata()
  error?: shared.ErrorT;

  @SpeakeasyMetadata()
  fileContent?: Uint8Array;

  @SpeakeasyMetadata()
  statusCode: number;
}

File: SDKs/apicurio.local/registry/2.1.0-SNAPSHOT/python/src/sdk/models/operations/exportdata.py
import dataclasses
from typing import Optional
from ..shared import error as shared_error


@dataclasses.dataclass
class ExportDataResponse:
    content_type: str = dataclasses.field()
    status_code: int = dataclasses.field()
    error: Optional[shared_error.Error] = dataclasses.field(default=None)
    file_content: Optional[bytes] = dataclasses.field(default=None)
    
